### PR TITLE
Chat: Fix contrast and colors of send button

### PR DESCRIPTION
**Pull request by toolmantim**
Fixes the send button styles, including hover, focus and high contrast modes:

https://github.com/sourcegraph/cody/assets/153/44e7a62e-1478-400e-8a9d-c9b46cbe919e

Also fixes the Storybook theme selector to apply the correct class name for the High Contrast Dark theme.

## Test plan

- CI

### CODE REVIEW

**Review by valerybugakov — APPROVED** [review]
🤌🧑‍🍳

**Comment by valerybugakov** [comment]
@toolmantim, merging your PR to include in the patch release

File: vscode/webviews/chat/cells/messageCell/human/editor/toolbar/SubmitButton.tsx [hunk 1]
```diff
@@ -30,9 +30,8 @@ export const SubmitButton: FunctionComponent<{
             disabled={disabled ? true : undefined}
         >
             {/* biome-ignore lint/a11y/noSvgWithoutTitle: */}
-            <svg width="20" height="20" viewBox="0 0 20 20">
-                <circle cx="10" cy="10" r="9.5" />
-                <path d="M8.25 6L14.25 10L8.25 14V6Z" strokeLinecap="round" strokeLinejoin="round" />
+            <svg width="8" height="10" viewBox="0 0 8 10">
+                <path d="M1.25 1L7.25 5L1.25 9V1Z" stroke-linecap="round" stroke-linejoin="round" />
```
Comment (keegancsmith) [inline vscode/webviews/chat/cells/messageCell/human/editor/toolbar/SubmitButton.tsx:34]: @toolmantim in CI output I am seeing complaints about `stroke-linecap`. What is interesting though is I see it defined this way inside of SVG files as well, which I assume where not handcrafted. So not sure if its just some CI noise?

```
Warning: Invalid DOM property `stroke-linecap`. Did you mean `strokeLinecap`?
    at path
    at svg
    at button
    at SubmitButton (/home/runner/work/cody/cody/vscode/webviews/chat/cells/messageCell/human/editor/toolbar/SubmitButton.tsx:7:25)
    at menu
    at Toolbar (/home/runner/work/cody/cody/vscode/webviews/chat/cells/messageCell/human/editor/toolbar/Toolbar.tsx:20:3)
    at div
    at HumanMessageEditor (/home/runner/work/cody/cody/vscode/webviews/chat/cells/messageCell/human/editor/HumanMessageEditor.tsx:16:3)
    at div
    at div
    at Cell (/home/runner/work/cody/cody/vscode/webviews/chat/cells/Cell.tsx:10:3)
    at BaseMessageCell (/home/runner/work/cody/cody/vscode/webviews/chat/cells/messageCell/BaseMessageCell.tsx:5:28)
    at HumanMessageCell (/home/runner/work/cody/cody/vscode/webviews/chat/cells/messageCell/human/HumanMessageCell.tsx:14:3)
    at TranscriptInteraction (/home/runner/work/cody/cody/vscode/webviews/chat/Transcript.tsx:81:16)
    at Transcript (/home/runner/work/cody/cody/vscode/webviews/chat/Transcript.tsx:17:23)
    at ClientActionListenersContextProvider (/home/runner/work/cody/cody/vscode/webviews/client/clientState.tsx:18:3)
    at Provider (file:///home/runner/work/cody/cody/node_modules/.pnpm/@radix-ui+react-context@1.0.1_@types+react@18.2.79_react@18.2.0/node_modules/@radix-ui/react-context/dist/index.mjs:41:28)
    at $a093c7e1ec25a057$export$f78649fb9ca566b8 (file:///home/runner/work/cody/cody/node_modules/.pnpm/@radix-ui+react-tooltip@1.0.7_@types+react@18.2.79_react-dom@18.2.0_react@18.2.0/node_modules/@radix-ui/react-tooltip/dist/index.mjs:42:29)
    at AppWrapper (/home/runner/work/cody/cody/vscode/webviews/AppWrapper.tsx:9:23)
    at TestAppWrapper (/home/runner/work/cody/cody/vscode/webviews/AppWrapper.tsx:21:27)
```

eg https://github.com/sourcegraph/cody/actions/runs/9351122980/job/25736111692?pr=4440#step:9:924

Comment (toolmantim) [reply vscode/webviews/chat/cells/messageCell/human/editor/toolbar/SubmitButton.tsx:34]: Sorry yep! Will fix.

**Pull request merged**